.github/workflows/pr_test_helm_chart.yml

```
@@ -0,0 +1,78 @@
+name: Test Helm Charts on KOF
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - opened
+      - synchronize
+      - reopened
+    branches:
+      - main
+      - release-*
+    paths:
+      - "promxy-operator/**"
+      - "charts/**"
+      - "!**.md"
+
+env:
+  GO_VERSION: "1.23"
+
+jobs:
+  test-and-deploy:
+    name: Test and Deploy KOF & KCM
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR ref
+        uses: actions/github-script@v7
+        id: pr
+        with:
+          script: |
+            const { data: pullRequest } = await github.rest.pulls.get({
+              ...context.repo,
+              pull_number: context.payload.pull_request.number,
+            });
+            return pullRequest
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{fromJSON(steps.pr.outputs.result).merge_commit_sha}}
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: false
+      - name: Unit tests for Promxy Operator
+        working-directory: ./promxy-operator
+        run: |
+          make test
+      - name: Create Kind Cluster
+        uses: helm/kind-action@v1.8.0
+        with:
+          version: v0.23.0
+          cluster_name: kof-cluster
+      - name: Checkout KCM repository
+        uses: actions/checkout@v4
+        with:
+          repository: k0rdent/kcm
+          ref: main
+          path: kcm-repo
+      - name: Install KCM CLI
+        run: |
+          make -C kcm-repo cli-install
+      - name: Apply KCM Dev Configuration
+        run: |
+          make -C kcm-repo dev-apply
+      - name: Install KOF CLI
+        run: |
+          make cli-install 
+      - name: Push KOF Helm Charts
+        run: |
+          make helm-push
+      - name: Deploy KOF Components
+        run: |
+          make dev-operators-deploy
+          make dev-ms-deploy-cloud
```

This command was renamed [today](https://github.com/k0rdent/kof/blob/8bb2dd94afe6664e838f5d28e7db2fc67df14c28/Makefile#L131):
```suggestion
          make dev-ms-deploy
```